src/library/export/dlglibraryexport.cpp

```diff
@@ -18,7 +18,7 @@
 #include "library/trackset/crate/cratestorage.h"
 #include "moc_dlglibraryexport.cpp"
 
-namespace el = djinterop::enginelibrary;
+namespace el = djinterop::engine;
```

Comment:
el isn't self explaining anymore, if you remove the word library

Reply:
Good spot - renamed to `e`.